nfsstat/tests/common.py

```diff
@@ -0,0 +1,41 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+from datadog_checks.dev import get_here
+
+COMPOSE_FILE = os.path.join(get_here(), 'compose', 'compose.yaml')
+
+CONFIG = {
+    "init_config": {"nfsiostat_path": "docker exec nfs-client /usr/sbin/nfsiostat"},
```

Comment:
```suggestion
```
You should be able to remove that, nfsiostat-systat is already packaged with the agent in the default location: `/opt/datadog-agent/embedded/bin//nfsiostat-sysstat`

Reply:
Can't mount nfs filesystems in the agent container, see https://github.com/DataDog/integrations-core/pull/6652#discussion_r426473271